Cargo.toml

```diff
@@ -19,7 +19,7 @@ members = [
 ]
 
 [workspace.package]
-version = "0.1.2"
```

Comment:
Should we also update the `[workspace.dependencies]` table? I guess it's not strictly needed since cargo will use the latest semver-compatible version anyway...

Reply:
Yes sorry!